lib/index.js

```diff
@@ -1,17 +1,17 @@
 var Promise = require('bluebird');
 var dgram = require('dgram');
 
-function Sitemon(type, hosts) {
-    this._type = type;
+function Sitemon(table, hosts) {
+    this._table = table;
     this._hosts = Array.isArray(hosts) ? hosts : [hosts];
     this._totalHosts = this._hosts.length;
     this._currentHostIndex = 0;
     this._client = dgram.createSocket("udp4");
 }
 
-Sitemon.prototype.send = function(data) {
+Sitemon.prototype.send = function(keys, values) {
```

Comment:
These are the nouns used within sitemon. The keys are key-value pairs where the keys are the column names and the values are the keys that get put into the key tables. They are only referenced in the table we log to by a foreign key ID.

The values are values that sitemon will sum together / aggregate per minute. Thus they have to be numbers. The keys again are the column names while the values are integers that will be summed up.

What would you prefer to call them?